docs/development/deploying-locally/index.md

```diff
@@ -1,3 +1,50 @@
 # Deploying locally
 
 REANA can be easily deployed locally on your laptop using Minikube.  Useful for REANA platform development.
+
+### Instructions
+
+* (1) Clone the main REANA repository:
+```console
+$ git clone https://github.com/reanahub/reana.git
+$ cd reana
+```
+
+* (2) Create a REANA development setup as follows:
+```console
+$ GITHUB_USER=JOHNDOE make setup clone prefetch
+```
+
+* (3) Build all components:
+```console
+$ make build
+```
+
+
+* (4) Deploy REANA:
+```console
+$ make deploy
+```
+
+Note that, REANA can also be deployed in development mode with code reload and [debugging capabilities](../debugging)
+```
+$ CLUSTER_FLAGS="debug.enabled=true" make deploy
```

Comment:
I wonder whether we shouldn't talk about CLUSTER_FLAGS only after the make variables, as an example of what can be done.

You can also mention "if you don't need UI, then you can use `EXCLUDE_COMPONENTS=r-ui,r-m-broker make ci` to speed up the process" etc etc.

So we'll have two nice examples after listing variables.